ui/client/components/Process/ProcessBackButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import {ProcessLink} from "../../containers/processLink"
+import {ButtonWithFocus} from "../withFocus"
+import {ReactComponent as ProcessBackIcon} from "../../assets/img/arrows/back-process.svg"
+import {useTranslation} from "react-i18next"
+
+type Props = {
+  processId: string,
+  processName?: string,
+}
+
+export default function ProcessBackButton(props: Props) {
+  const {processId, processName} = props
+  const process = processName || processId
+  const {t} = useTranslation()
+
+  return (
+    <div className={"back-to-process-button"}>
+      <ProcessLink processId={processId}>
+        <ButtonWithFocus className="back-to-process-button">
+          <ProcessBackIcon/>
+          <span className="button-text">
+            {t("processBackButton.text", "back to process {{process}}", {process})}
```

Comment:
Back to process {{...?

Reply:
On mock was small letter.. ;) But I've changed it to big one.